frontend/uddhar-frontend/src/authentication/components/OrganizationDetails.jsx

```
@@ -0,0 +1,329 @@
+      <div className="flex flex-col mb-4">
+        <label htmlFor="mission">Mission of the Organization</label>
+        <textarea
+          name="mission"
+          id="mission"
+          cols="50"
+          rows="3"
+          className={`resize-none border p-3 my-5 ${
+            errors.mission ? "border-red-500" : "border-gray-300"
+          }`}
+          onChange={handleChange}
+        ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)